win32ss/user/user32/misc/imm.c

```diff
@@ -720,8 +737,8 @@ static LRESULT ImeWnd_OnImeSystem(PIMEUI pimeui, WPARAM wParam, LPARAM lParam)
             ret = IMM_FN(ImmFreeLayout)((HKL)lParam);
             break;
 
-        case 0x13:
-            FIXME("\n");
+        case IMS_SOFTKBDONOFF:
+            IMM_FN(ImmEnumInputContext)(0, ImeWnd_SwitchSoftKbdProc, lParam);
```

Comment:
This doesn't return an error code?

Reply:
`ImmEnumInputContext` will trace on error. No need to check return value.